src/docs/files/devguide02.md

```diff
@@ -2,36 +2,16 @@
 
 1. In batch auctions, there is one auctioneer and multiple bidders.
 
-2. The auctioneer selects the amount of an asset they want to sell and determine the minimum price they are willing to receive for the asset
+2. The auctioneer selects the amount of an asset they want to sell and determine the minimum price they are willing to receive for the asset/s.
 
 3. The auction begins, and it is open for a specified duration. Bidders are free to place their bids, through limit orders, once the auction starts.
 
 4. After the specified duration is over, the auction ends, and no more bids can be placed. 
 
-5. The smart contract sorts bids from highest to lowest price. Bids include a limit order for the token price and the amount of tokens to buy.
+5. The smart contract sorts bids from highest to lowest price. Bids include a limit order for the token price and the amount of tokens to sell.
 
 6. The smart contract works backward from the highest bid, cumulatively adding each bid’s amount of tokens to sell, until the bid's sum multiplied by the price of the highest bid equals the sell amount of tokens (pre-defined by the auctioneer).
 
 7. The price of the bid (added to all previous bids’ amount of tokens) that reaches the amount of tokens to sell is selected as the final clearing price for all participants. 
 
-8. Bidders who specified a maximum price in their limit order that is equal to or greater than the final clearing price receive tokens at the clearing price. 
-
-9. Bidders that specified a maximum price in their limit order that is less than the final clearing price do not receive the tokens being auctioned, but they can still withdraw their bidding tokens from the DEX. 
-
-Let's take a deeper look at the example provided in the diagram below.
-
-
-The following diagram shows an auction in which 14 tokens are offered at 2 blocks per share:
-
```

Comment:
do you wanna delete all of that without substitution?

Reply:
yes, this is part of the announcement post that I added without checking in with @Rafanator. We agreed that it would be better to keep the section as simple as possible as it was created originally. Maybe, if possible, its best if we wait for tomorrow to approve this pull request so @Rafanator can weigh in.

Reply:
I think we should at least that something like that:

8. Every user that had a limit price higher than the clearing price can claim the auctioned tokens priced at the clearing price. All other users with orders lower that the clearing price will claim their initial investments back...

Reply:
Although this was not in the original section, I think it can help clarify an important part without needing to go to the FAQ section.

Reply:
What is the current sentiment? I felt like I needed to explain it too so many people who really wanted to know what its going on that I defintively votte for the longer version

Can we close this. PR/

Reply:
If you both think it is better, let's do it.